examples/dev-playground/README.md

```diff
@@ -16,19 +16,19 @@
 
 #### Step 1.
 
-```sh
-yarn install
+copy `.env.template` to `.env` and add the required variables. You can use this to generate a string for `ETHEREUM_JWT_SECRET`:
```

Comment:
It makes more sense to set-up the environment variables before doing `yarn prisma migrate` etc.